shortcodes/class.map-shortcode.php

```diff
@@ -317,6 +318,22 @@ protected function getHTML($atts='', $content=null)
             echo do_shortcode('[leaflet-scale noScriptWrap]');
         }
 
+        echo '
+          window.WPLeafletMapPlugin = window.WPLeafletMapPlugin || [];
+          window.WPLeafletMapPlugin.push(function () {
+            var map = window.WPLeafletMapPlugin.getCurrentMap();
+            map.eachLayer(function(layer) {
+              if( layer instanceof L.TileLayer ) {
+                if ( map.options.minZoom < layer.options.minZoom) {
+                  map.options.minZoom = layer.options.minZoom;
+                }
+                if (map.options.maxZoom > layer.options.maxZoom) {
+                  map.options.maxZoom = layer.options.maxZoom;
+                }
+              }
+            });
+          });
+        ';
```

Comment:
I have no idea what this is, but it doesn't look right

Reply:
I tried to prevent that the zoom of the map greater or smaller is as the possible tiles zoom.